Sources/Sentry/include/SentryTraceContext.h

```diff
@@ -50,7 +50,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * Sample rate used for this trace.
  */
-@property (nullable, nonatomic) NSString *sampleRate;
+@property (nullable, nonatomic, strong) NSString *sampleRate;
```

Comment:
All the other string properties are `readonly` so they don't need a retain attribute since no objects will be set into them from outside this class. Is this property different than those? I don't see any assignments to this property.

Reply:
SampleRate cant be readonly because we only now the value used for this after the SentryTraceContext being used to initialize the transaction.